mono/metadata/icall-windows.c

```diff
@@ -45,25 +45,26 @@ mono_icall_module_get_hinstance (MonoReflectionModuleHandle module)
 	return (gpointer) (-1);
 }
 
-#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT) || G_HAVE_API_SUPPORT(HAVE_UWP_WINAPI_SUPPORT)
```

Comment:
I think, you would prefer, this identical content be in icall-windows-uwp.c, instead of icall-windows.c? Identical content and identical if, still used by both classic and uwp?